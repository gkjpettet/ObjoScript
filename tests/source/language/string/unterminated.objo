// Expect error.
"this string has no closing quote