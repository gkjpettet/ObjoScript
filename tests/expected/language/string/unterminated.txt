Unterminated string literal. Expected a closing double quote.